src/ASTInterpreter2.jl

```diff
@@ -12,8 +12,12 @@ using Markdown
 
 export @enter, @make_stack
 
-struct CustomType end
-# include("linearize.jl")
+struct Wrap{V}
```

Comment:
Just use `Some` and `something` from base.

Reply:
Ah okay!